build.gradle

```diff
@@ -3,7 +3,7 @@ apply plugin: 'maven'
 apply plugin: 'signing'
 
 group = 'com.cloudant'
-version = '1.2.0'
+version = '1.2.1-SNAPSHOT'
```

Comment:
We didn't have a maintenance branch when I made this branch, but then I added the commit to the maintenance branch and I thought it had vanished from this one.